test/io/test_load_image.py

```diff
@@ -22,7 +22,7 @@ def available_package() -> bool:
 
 
 def create_random_img8(height: int, width: int, channels: int) -> np.ndarray:
-    return (np.random.rand(height, width, channels) * 255).astype(np.uint8)
+    return (np.random.Generator(height, width, channels) * 255).astype(np.uint8)
```

Comment:
NPY002	[ ] Replace legacy `np.random.rand` call with `np.random.Generator`

% `ruff rule NPY002`
# numpy-legacy-random (NPY002)

Derived from the **NumPy-specific rules** linter.

## What it does
Checks for the use of legacy `np.random` function calls.

## Why is this bad?
According to the NumPy documentation's [Legacy Random Generation]:

> The `RandomState` provides access to legacy generators... This class
> should only be used if it is essential to have randoms that are
> identical to what would have been produced by previous versions of
> NumPy.

The members exposed directly on the `random` module are convenience
functions that alias to methods on a global singleton `RandomState`
instance. NumPy recommends using a dedicated `Generator` instance
rather than the random variate generation methods exposed directly on
the `random` module, as the new `Generator` is both faster and has
better statistical properties.

See the documentation on [Random Sampling] and [NEP 19] for further
details.

## Examples
```python
import numpy as np

np.random.seed(1337)
np.random.normal()
```

Use instead:
```python
rng = np.random.default_rng(1337)
rng.normal()
```

[Legacy Random Generation]: https://numpy.org/doc/stable/reference/random/legacy.html#legacy
[Random Sampling]: https://numpy.org/doc/stable/reference/random/index.html#random-quick-start
[NEP 19]: https://numpy.org/neps/nep-0019-rng-policy.html

Reply:
test/io/test_load_image.py:25:12: NPY002 Replace legacy `np.random.uniform` call with `np.random.Generator`
Found 1 error.